context-free.md

```diff
@@ -1,22 +1,37 @@
 # Context Free
 
 ##  Who is this programming language for?
-
+This programming language is for people who want to create digital art.
 
 ## What is easy to do in this language? Why is it easy?
-
+Recursively creating shapes is easy because shapes are created inside rules.
 
 ## What is hard to do in this language? Why is it hard?
-
+It was very hard for me to learn it, taking well over an hour.  It requires
+memorization of syntax and avaiable rules unlike a simple program such as paint
+where you can see what is available for use.  Additional memorization is
+required to know what transforamtions you have available for your shape (such
+as using hue to color things is less obvious than something like color). Drawing
+like one would with a pencil is quite hard because everything has to be
+programmed in and cannot be drawn using mouse.
 
 ## How did you learn how to program in this language?
-
+Struggled with the documentation and looking at the sample code so I found a
+youtube video tutorial, which explained things well but for the incorrect
+version.  However after the youtube tutorial, the documentation made more sense.
```

Comment:
The version issue is definitely a huge pain point with this language, as is the documentation. I didn't try any youtube tutorials, but if you have a link I'm sure @bwiedermann would appreciate having that resource for future classes!